src/mpi/romio/adio/common/lock_internal.c

```diff
@@ -0,0 +1,44 @@
+#include "lock_internal.h"
+
+const char *flock_cmd_to_string(int cmd)
```

Comment:
This still has a namespace issue.  You are exposing symbols that are not in the mpich reserved set of namespaces, i.e., `MPIR`, `MPID`, or `ADIO`.